cmd/start.go

```diff
@@ -107,6 +117,17 @@ func buildCRWatcher(cfg *restclient.Config) *crwatcher.CRWatcher {
 	return crw
 }
 
+func getController() crwatcher.ResourceController {
```

Comment:
If a user specified both flags this would silently use helm rather than tmplctlr.  Should we log that we're using the helm controller, or fail if both sets are specified?  Or would subcommands under start make sense?

Reply:
As part of #1 I'll address adding more details around this and adding failures if you specify flags for both code paths.